phpcs.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<ruleset name="Project Rules">
+	<rule ref="10up-Default">
+		<exclude name="PEAR.NamingConventions.ValidClassName.StartWithCapital" />
+		<exclude name="PEAR.NamingConventions.ValidClassName.Invalid" />
+		<exclude name="WordPress.Files.FileName.InvalidClassFileName" />
+	</rule>
+
+	<exclude-pattern>*/tests/*</exclude-pattern>
```

Comment:
```suggestion
	<exclude-pattern>*/dist/*</exclude-pattern>
	<exclude-pattern>*/node_modules/*</exclude-pattern>
	<exclude-pattern>*/tests/*</exclude-pattern>
	<exclude-pattern>*/vendor/*</exclude-pattern>
```

A few additional folders to ignore: mainly to save time but the `dist` folder will throw up some false positives due to generated files.

Reply:
Fixed here: https://github.com/10up/insecure-content-warning/pull/127/commits/5c9e19fc5bd372d492877fd0bdbd79ea0a790038